arcgis-ios-sdk-samples/Search/Reverse geocode/README.md

```diff
@@ -1,17 +1,33 @@
 # Reverse geocode
 
-This sample demonstrates how to get the address for a point on the map
+Use an online service to find the address for a tapped point.
 
-## How to use the sample
+![Reverse geocode sample](reverse-geocode.png)
+
+## Use case
 
-Tap on a point on the map, a graphic will be added at that point and a callout will show the address for that point
+You might use a geocoder to find a customer's delivery address based on the location returned by their device's GPS.
 
-![](image1.png)
+## How to use the sample
 
+Tap the map to see the nearest address displayed in a callout.
 
 ## How it works
 
-The sample uses the `reverseGeocode(withLocation:parameters:completion:)` method on `AGSLocatorTask` to find an address for a point. To show the callout the sample uses the `show(for:tapLocation:animated:)` method on `AGSCallout`.
+1. Create an `AGSLocatorTask` object using a URL to a geocoder service.
+2. Set the `AGSGeocodeParameters` for the `AGSLocatorTask` and specify the geocoder's attributes.
+3. Get the matching results from the `AGSGeocodeResult` using  `AGSLocatorTask.reverseGeocode(withLocation:parameters:completion:)`.
```

Comment:
Again, the parameters object is passed as a parameter to the reverse geocode method, not set on the locator task.